apps/payments/next/app/[offeringId]/checkout/page.tsx

```diff
@@ -22,26 +22,30 @@ export default async function Index({ params }: { params: CheckoutParams }) {
     <>
       <h1 className="page-title-container">Under Construction</h1>
       <section className="payment-panel" aria-label="Purchase details">
-        <PurchaseDetails
-          interval={cart.interval}
-          invoice={cart.nextInvoice}
-          purchaseDetails={contentful.purchaseDetails}
-        />
+        <div className="component-card text-sm px-4 rounded-t-none tablet:rounded-t-lg">
```

Comment:
Moved the container styling out of `<PurchaseDetails>` so that the consumer, the checkout page in this case, can decide how to style it. Happy to move it back into `<PurchaseDetails>` though.

Reply:
My preference would be to keep this styled `div` within the component, so the basic outline of this section on the Checkout page looks as follows:
```
<section>
   <PurchaseDetails />
   <CouponForm />
</section>
```
as opposed to:

```
<section>
  <div>
     <PurchaseDetails />
  </div>
  <div>
    <CouponForm />
  </div>
</section>
```

Reply:
Fully aligned with Lisa.
- The extra `div` enclosures are unnecessary. Further, the more embedded content is within excessive `divs`, the harder it is for ATs to reach.
- Placement of the components will be determined by the surrounding `section`.
- Each component will have minimum custom styling as the content is relatively sparse and mostly copy and `form` elements, both of which will have shared styles (e.g. `font-family`, `font-size`, `form` element styles, etc.).